src/configure_edison

```diff
@@ -189,12 +189,13 @@ def _getBTMode():
     return "Off"
 
 def _getGadgetMode():
-    modestr = ''
-    modestr = subprocess.run('lsusb', shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    modestr = str(modestr.stderr, 'utf-8')
-    gadget = modestr.find("unable to initialize libusb")
-    if gadget == -1:
+    try:
+        subprocess.check_output('dmesg | grep -q "usb0: link becomes ready"', shell=True)
```

Comment:
As it was, you could change gadget <-> host mode on the fly and the web page page would reflect that (wired = gadget, wired = dongle or wired = none).

Reply:
I just asked @andy-shev and he says: 'Non-reliable (in long-term) is to use debugfs I think'. 

Still I think there should be something hidden under /sys or so reflecting the state.

Reply:
Unfortunately I didn't find anything. Perhaps you make ask in linux-usb@ mailing list how user space is supposed to detect the mode.

Reply:
Alright

Reply:
Okay, but I think we could merge the PR and sort that out later (the dmesg solution is not perfect, but it works for the first time you plug-in).

Reply:
I think we may do it like this:

- in gadget mode `lsusb` returns nothing
- in host mode it returns minimally root hub

In my case:
```
root@edison:~# lsusb
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 004: ID 125f:312b A-DATA Technology Co., Ltd. Superior S102 Pro
Bus 001 Device 003: ID 0424:ec00 Standard Microsystems Corp. SMSC9512/9514 Fast Ethernet Adapter
Bus 001 Device 002: ID 0424:9514 Standard Microsystems Corp. SMC9514 Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```

Reply:
Okay, so the `check_output` argument may be replaced with `lsusb -d 1d6b:0003` (if the Linux Foundation 3.0 root hub is there or not).